nixos/tests/terminal-emulators.nix

```
@@ -197,7 +197,8 @@ in mapAttrs (name: { pkg, executable ? name, cmd ? "SHELL=$command ${executable}
 
     with subtest("have the terminal display a colour"):
         # We run this command in the background
-        machine.shell.send(b"(run-in-this-term display-colour |& systemd-cat -t terminal) &\n")
+        if machine.shell is not None:
+            machine.shell.send(b"(run-in-this-term display-colour |& systemd-cat -t terminal) &\n")
```

According to [this](https://engineeringblog.ridereport.com/pythons-none-problem) this is another way to do it.
It does not suggest this command to be optional somehow.

```suggestion
        assert machine.shell is not None
        machine.shell.send(b"(run-in-this-term display-colour |& systemd-cat -t terminal) &\n")
```